stl/inc/xutility

```diff
@@ -1953,7 +1957,10 @@ public:
     }
     // clang-format on
 #else // ^^^ __cpp_lib_concepts / !__cpp_lib_concepts vvv
-    _NODISCARD _CONSTEXPR17 pointer operator->() const {
+    _NODISCARD _CONSTEXPR17 pointer operator->() const
+        noexcept(is_nothrow_copy_constructible_v<_BidIt>&& noexcept(--(_STD declval<_BidIt&>())) && noexcept(
+            _Implicitly_convert_to<pointer>(
+                _Operator_arrow(_STD declval<_BidIt&>(), is_pointer<_BidIt>())))) /* strengthened */ {

```

Comment:
The tests are failing with:
```
D:\build\x64\out\inc\xutility(1963): error C3861: '_Operator_arrow': identifier not found
C:\a\1\s\tests\std\tests\P0220R1_string_view\test.cpp(1096): note: see reference to function template instantiation 'bool test_case_iterators<char,std::char_traits<char>>(void)' being compiled
```

Reply:
A sorry, I propared this during the ranges frenzy and forgot about it. I should have checked whether it still applies after the rebas.